src/client/components/RegistrationConsents.tsx

```diff
@@ -25,11 +27,28 @@ export const RegistrationConsents = ({
 	geolocation,
 	useIdapi,
 	noMarginBottom,
+	appName,
 }: RegistrationConsentsProps) => {
-	// don't show the Saturday Edition newsletter option for US and AUS
-	const showSaturdayEdition = !(['US', 'AU'] as GeoLocation[]).some(
-		(location: GeoLocation) => location === geolocation,
-	);
+	// don't show the Saturday Edition newsletter option for US and AUS or if using Feast app
+	const showSaturdayEdition = (() => {
+		const isValidLocation = !(['US', 'AU'] satisfies GeoLocation[]).some(
+			(location: GeoLocation) => location === geolocation,
+		);
+
+		const isValidAppName = !(['Feast'] satisfies AppName[]).some(
```

Comment:
This is the same check as on line 46 - could we just have the one check?

Reply:
Changed it a little

added

```ts
	// check if the app is Feast or not
	const isFeast = appName === 'Feast';
```

so the checks respectively become

saturday edition
```ts
return isValidLocation && !isFeast;
```

feast
```ts
	// show the Feast newsletter option if the app is Feast and Saturday Edition is not shown
	const showFeast = isFeast && !showSaturdayEdition;
```

Reply:
Much nicer!